lib/model.js

```diff
@@ -3385,10 +3385,10 @@ class Model {
 
     return promise.then(affectedRows => {
       if (options.returning) {
-        return [affectedRows, affectedRows.length];
+        return [affectedRows[0], affectedRows[1]];
```

Comment:
You could just return the tuple directly 😛

Reply:
Done :smile: